src/remixAppManager.js

```diff
@@ -214,6 +214,18 @@ export class RemixAppManager extends PluginEngine {
       version: '0.1.0-beta',
       location: 'sidePanel'
     }
+    const remixdgit = {
+      name: 'Remixd.git',
```

Comment:
I'm not sure the bame can contain '.'
@GrandSchtroumpf ?

Reply:
Will remove it completely :)